### PR TITLE
added status codes enum

### LINKED ISSUE #220: HTTP Status Codes
Hi, 

I would appreciate some defined constants for the http status codes. I couldn't find any.

### PR DESCRIPTION
**Pull request by The-EDev**
closes #220

### CODE REVIEW

**Review by luca-schlecker — DISMISSED** [review]
Looking good. 👍

File: include/crow/http_connection.h [hunk 1]
```diff
@@ -418,6 +418,7 @@ namespace crow
                 return;
             }
 
+            // Keep in sync with common.h/status
             static std::unordered_map<int, std::string> statusCodes = {
                 {100, "HTTP/1.1 100 Continue\r\n"},
```
Comment (mrozigor) [inline include/crow/http_connection.h:423]: Maybe change status members to enum entries?

Comment (The-EDev) [reply include/crow/http_connection.h:423]: Could you elaborate please?

Comment (luca-schlecker) [reply include/crow/http_connection.h:423]: Instead of mapping an `int` to a `string`, he suggests directly mapping the enum type `status` to a `string`.

Comment (The-EDev) [reply include/crow/http_connection.h:423]: hmmmm, I'll test that approach

Comment (The-EDev) [reply include/crow/http_connection.h]: done, all tests ran fine and anyone can still use integers or the enum as the response code.

**Review by mrozigor — APPROVED** [review]

**Pull request merged**